docs/lib/datastore/fragments/android/getting-started/50_initDataStore.md

```diff
@@ -0,0 +1,66 @@
+To initialize the Amplify Storage and Authentication categories, we are required to use the `Amplify.addPlugin()` method for each category we want.  When we are done calling `addPlugin()` on each category, we finish configuring Amplify by calling `Amplify.configure()`.
+
+
+Add the following code to the bottom of your MainActivity `onCreate` method (ideally this would go in your Application class but this works for getting started quickly):
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.addPlugin(new AWSDataStorePlugin());
+```
+
+Your class will look like this:
+
+```java
+public class MyAmplifyApp extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        try {
+            Amplify.addPlugin(new AWSDataStorePlugin());
```

Comment:
Do we need API here?

Reply:
Actually no, in iOS I only add it later. I'll update.